src/schoology-extractor/schoology_extractor/helpers/sync.py

```diff
@@ -72,15 +72,18 @@ def _resource_has_changed(
     db_parsed_item = json.loads(json_response)
 
     for key in db_parsed_item:
-        api_value = resource[key]
-        db_value = db_parsed_item[key]
+        ignore_key = key == 'CreateDate' or key == 'LastModifiedDate'
 
-        if isinstance(api_value, dict) or isinstance(api_value, list):
-            api_value = str(api_value).replace("'", '"')
-            db_value = str(db_value).replace("'", '"')
+        if not ignore_key:
```

Comment:
To reduce the level of nesting and make this a little more readable, it may be useful to change this to:

```python
if ignore_key:
    continue
```

Reply:
✔️